### PR TITLE
feat: Machine translation and Translation memory docs

### PR DESCRIPTION
**Pull request by stepan662**

### CODE REVIEW

File: docs/platform/self_hosting/configuration.mdx [hunk 1]
```diff
@@ -397,6 +397,19 @@ While this is useful for tests, make sure to keep this **disabled** if you're im
 server, as trying this predictable key may be the first thing an attacker will attempt to gain unauthorized access!
 :::
 
+## Machine translation
+Prefix: `tolgee.machine-translation`
+
+### free-credits-amount
+Amount of translating credits that user can use per month. (unlimited if not specified)
+
+### google.api-key
```
Comment (JanCizmar) [inline docs/platform/self_hosting/configuration.mdx:406]: Each of the service has also option `default-enabled` and `default-primary`.

File: docs/platform/self_hosting/configuration.mdx [hunk 2]
```diff
@@ -397,6 +397,19 @@ While this is useful for tests, make sure to keep this **disabled** if you're im
...
+Your Google api key, which enables Google provider.
+
+### aws.accessKey and aws.secretKey
```
Comment (JanCizmar) [inline docs/platform/self_hosting/configuration.mdx:409]: `aws.access-key` and `aws.secret-key`

File: docs/platform/translation_tools.mdx [hunk 3]
```diff
@@ -0,0 +1,29 @@
+---
+id: translation_tools
+title: Translation tools
+sidebar_label: Translation tools
+slug: /platform/translation_tools
+---
+
+Translation tools helps you by suggestions based on base translation.
```
Comment (JanCizmar) [inline docs/platform/translation_tools.mdx:8]: help

**Review by JanCizmar — APPROVED** [review]

**Comment by JanCizmar** [comment]
Ahh, my fault. Yep, It can be reused.

**Comment by stepan662** [comment]
Thanks @cyyynthia , I've completely missed it 😄  Yep I'll reuse it-

**Pull request merged**